index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Zilola's portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header id="home-header">
+      <nav id="toolbar-container">
+        <a href="#" class="logo-placeholder">My Logo</a>
+        <button class="burger"></button>
+      </nav>     
+    </header>
+    <main id="home-main">
+      <section id="headline">
+        <h1 class="headline-item" id="greeting">I’m Alexandra.<br /> Glad to see you!</h1>
```

Comment:
- [OPTIONAL] Kindly consider personalizing by replacing `Alexandra` with your name. because it is your portfolio and you need to personalize it.